src/rpcwallet.cpp

```diff
@@ -2684,6 +2689,12 @@ Value exportzerocoins(const Array& params, bool fHelp)
         denomination = libzerocoin::IntToZerocoinDenomination(params[1].get_int());
     list<CZerocoinMint> listMints = walletdb.ListMintedCoins(!fIncludeSpent, false, false);
 
+    for(CZerocoinMint mint : listMints) {
```

Comment:
Seems like bad form to have two identical for loops right next to each other. Any particular reason not to decrypt the mint within the loop a few lines below?

Reply:
The loop below was const objects. Wasn't sure if there is reasoning around when to treat the objects as constants and when to treat them as mutable. Is there any particular reason in the loop below it is marked const?